tests/core/v5_1/alexandria/test_alexandria_client.py

```diff
@@ -131,6 +131,17 @@ async def test_alexandria_client_send_find_nodes(
         assert message.payload.distances == (0, 255, 254)
 
 
+@pytest.mark.trio
+async def test_alexandria_client_send_find_nodes_timeout(
+    alice, bob, bob_client, alice_alexandria_client, autojump_clock,
+):
+    with pytest.raises(trio.TooSlowError):
+        async with bob_client.dispatcher.subscribe(TalkRequestMessage) as subscription:
```

Comment:
I think you can drop this line, as well as drop out the entire `bob_client` from this test.